src/components/Homepage/LendByCategory/LoanCategory.vue

```diff
@@ -53,6 +59,14 @@
 					<div
 						class="column cards-wrap"
 					>
+						<!-- TODO:
+						---
+						---
+						This is the only piece of tracking that needs to be fixed
```

Comment:
What needs to be fixed about the view all card tracking?

Reply:
There's no indication that a click on this card is coming from the lend by category homepage.